admin/spec/controllers/group.spec.js

```diff
@@ -14,7 +14,7 @@ const groupsMock = require('../mocks/groups')
 const groupDeletedMock = require('../mocks/group-deleted')
 const pupilsMock = require('../mocks/pupils-with-reason')
 
-describe('group.js controller', () => {
+xdescribe('group.js controller', () => {
```

Comment:
I don't see any reason why the updated tests should not be supplied with the code changes.